Apps/APN/.gitignore

```
@@ -0,0 +1,30 @@
+app_store_connect_creds.json
+ami_app_ci_server-google_cloud_service_account.json
+gc_keys.json
+GoogleService-Info.plist
+node_modules
+.DS_Store
+env.js
+fastlane/report.xml
```

```suggestion
node_modules
.DS_Store
env.js
fastlane/report.xml
```

Some of these are outdated and can be removed since we don't use the files anymore. 

As for the GoogleService-Info.plist, in the iOS SDK sample apps, we are not ignoring this file. So I suggest we follow the same pattern in this repository.